src/extension.ts

```diff
@@ -59,9 +59,8 @@ function runJavaServer(context: VSCode.ExtensionContext): Thenable<StreamInfo> {
             requirements,
             server.address().port
           );
-          let options = { cwd: VSCode.workspace.rootPath };
           console.log("Executing " + command + " " + args.join(" "));
-          let process = ChildProcess.spawn(command, args, options);
+          let process = ChildProcess.spawn(command, args);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Identifier 'process' is never reassigned; use 'const' instead of 'let'. ([typescript:S3353](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS3353))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarlint.vscode%3Asonarlint-vscode&pullRequest=16&issues=AWMHuAuKqwcY0PgyxSYp&open=AWMHuAuKqwcY0PgyxSYp)

[ISSUE_KEY]: AWMHuAuKqwcY0PgyxSYp